lib/foo.js

```diff
@@ -2,7 +2,9 @@ var service = require('./service');
 
 exports.bar = function (context, event) {
   if (!event.size) {
+    console.log('Missing parameter: size');
     context.fail('Missing parameter: size');
+    return;
```

Comment:
Bingo! Sinon already has a spy.throw() method (no need for other framework). http://sinonjs.org/docs/